Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+gem 'sinatra'
+gem 'mechanize', '~>2.7.2', '<2.7.3'
+gem 'nokogiri'
+gem 'thin'
+
+group :development do
+  gem 'pry-byebug'
```

Comment:
I didn't do the group in the Gemfile, good call, I will have to read more about this >_<